modules/example/src/smithy4s/example/StructureWithRefinedMember.scala

```diff
@@ -4,19 +4,18 @@ import smithy4s.Hints
 import smithy4s.Schema
 import smithy4s.ShapeId
 import smithy4s.ShapeTag
-import smithy4s.example.refined.Age
 import smithy4s.example.refined.Age.provider._
 import smithy4s.schema.Schema.int
 import smithy4s.schema.Schema.struct
 
-case class StructureWithRefinedMember(otherAge: Option[Age] = None)
+case class StructureWithRefinedMember(otherAge: Option[smithy4s.example.refined.Age] = None)
```

Comment:
What caused this to no longer be imported at the top? I see a few others as well so maybe a regression of some kind

Reply:
fixed, in https://github.com/disneystreaming/smithy4s/pull/848/commits/f507ab267e75b54e6cc7a0b08cdf64a02fafd0ca

Reply:
The reason why the bug was not exhibited is that the compiler option added was incorrect. See :  

[f84f208](https://github.com/disneystreaming/smithy4s/pull/848/commits/f84f20813e8ce6229aed105127b0a76ec08326ae)

Reply:
> The reason why the bug was not exhibited is that the compiler option added was incorrect. See :

I dont think so , I tried that first and got ` bad option: '-source:3.0-migration'`

Reply:
I see this option only works with the Scala 3 compiler